### ISSUE TITLE
Mastodon Social Network Logo included on nextcloud.tld/settings/admin

### ISSUE THREAD

**Issue by ZiiX** [nextcloud/server#6714]
Any chance y'all could include a fellow open-sourced project, Mastodon, that's not participating in surveillance capitalism? This would be for personal profiles (in lieu of twitter) as well as the ../settings/admin page for your @mastodon@mastodon.xyz account. : P

![social icons](https://user-images.githubusercontent.com/2234740/31049743-7efe45a6-a5ee-11e7-9e99-3c2d3b31a8de.png)

tx <3

**Comment by nickvergessen** [comment]
@jospoortvliet

**Comment by nickvergessen** [comment]
Is https://mastodon.xyz/@nextcloud our official account?

**Comment by jospoortvliet** [comment]
It is! Yeah, we could add it.

**Comment by jancborchardt** [comment]
Yes, the @nextcloud@mastodon.xyz is official. @ZiiX mind opening a pull request to add it in there, right of Twitter?

**Comment by violoncelloch** [comment]
may I work on this?
@jancborchardt is there a resource for the icons or do we have to create one for mastodon first?

**Comment by violoncelloch** [comment]
i created an icon in the style of the others using mastodon's press kit:
![mastodon](https://user-images.githubusercontent.com/22591354/45229669-c39e4b80-b2c6-11e8-8a4b-68f433809a81.png)
any feedback?
